src/main/kotlin/com/pubnub/api/subscribe/eventengine/effect/ReceiveMessagesProvider.kt

```diff
@@ -4,7 +4,7 @@ import com.pubnub.api.endpoints.remoteaction.RemoteAction
 import com.pubnub.api.subscribe.eventengine.event.SubscriptionCursor
 
 fun interface ReceiveMessagesProvider {
-    fun receiveMessages(
+    fun getRemoteActionForReceiveMessages(
```

Comment:
`receiveMessagesRemoteAction`?

Reply:
Ok. I will change it to getReceiveMessagesRemoteAction.